internal/services/database/mysql/migrations.go

```diff
@@ -119,3 +120,11 @@ func migration_10(m *sql.Tx) (err error) {
 		"guilds", "`birthdaychanID` text NOT NULL DEFAULT ''")
 	return
 }
+
+// VERSION 11:
+// - add property `autovc` to `guilds`
+func migration_11(m *sql.Tx) (err error) {
+	err = createTableColumnIfNotExists(m,
+		"guilds", "`autovc` text NOT NULL DEFAULT ''")
+	return
+}

```

Comment:
This implementation only allows one auto voice channel for each guild. Would be better if we create a new table where all created auto VCs are listed in, so you can create multiple per guild.

The table would be like
| autovcs |
|---|
| iid `int` *(PK - auto increment)* |
| guildID `string` |
| chanID `string` |

Wouldn't include this as requirement for this PR and would later modify it, if you don't want to do it. ^^

Reply:
Setting multiple auto VCs should actually work.
When implementing the autovc feature, I closely followed the implementation of the `autorole` command.
There the roles get concatenated to a string separated by semicolons.
That said, a separate table for auto VCs might be the cleaner way to implement this.
Let me know what you think and I can implement it this way 😄

Reply:
Ah okay, sorry, I missed that. Then I would let the implementation this way for now. 

Sadly, my database implementation is very old and also extremely unoptimized and bad implemented. Maybe later I find the time to completely rework the database bindings, but that would also need a lot migration work. ^^